python-checks/src/test/resources/checks/hotspots/unsafeHttpMethods/django/views.py

```diff
@@ -49,3 +49,13 @@ def sensitive_other_dec(request): # Noncompliant
 
 def other(request):
   ...
+
+
+from django.urls import path
+
+class SameClassView:
+  def chosen_view(self):  # FN SONARPY-2322
+    ...
+
```

Comment:
_extra mile_ to get even closer to the original issue from our [analysis](https://docs.google.com/spreadsheets/d/1KvINxh1yjmz7nbI0-to9p96K6b8Oby4Eux7473ifNy0/edit?gid=1339885434#gid=1339885434), I would add a method using a property decorator :

```python
@property
def view_property(self): 
```
and ensure that we have also the FN.

Reply:
The rule already tests for various decorators. I deliberately removed the `@property` one so that it doesn't bring confusion as to the root cause of the FN.

Reply:
ok makes sense !